gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/handlers/api/ApiContextHandlerFactory.java

```diff
@@ -102,7 +105,10 @@ public ReactorHandler create(Api api) {
             if (api.isEnabled()) {
                 Class<?> handlerClass = this.getClass().getClassLoader().loadClass(ApiReactorHandler.class.getName());
 
-                final ApiReactorHandler handler = (ApiReactorHandler) handlerClass.getConstructor(Api.class).newInstance(api);
+                final ApiReactorHandler handler = (ApiReactorHandler) handlerClass
+                    .getConstructor(Api.class, Long.class)
+                    .newInstance(api, pendingRequestsTimeout);
+
```

Comment:
Do you think we can use the constructor directly instead of using the reflection ?